server/model/dbSetupQuery.js

```diff
@@ -0,0 +1,40 @@
+const users = `
+DROP TABLE IF EXISTS users cascade;
+CREATE TABLE users(
+  userId SERIAL PRIMARY KEY,
+  fullName VARCHAR(60) NOT NULL,
+  email VARCHAR(40) NOT NULL 
+);`;
+
+const authentication = `
+DROP TABLE IF EXISTS authentication cascade;
```

Comment:
Imagine a situation when you have a new table and you want to run migrations again, You would eventually lose al your tables using this step